dist/cloudfront/get.openziti.io/routes.yml

```diff
@@ -38,3 +38,7 @@
 - get: /zdew/
   raw: /openziti/desktop-edge-win/main/release-streams/
   file: latest.json
+
+- get: /dist/
```

Comment:
simplify the tproxy example by providing a short getter for the latest  ziti-router compose file, and anything else in `/dist/`.